cfme/tests/services/test_provision_stack.py

```diff
@@ -337,7 +177,6 @@ def _cleanup_vms():
             if provider.mgmt.stack_exist(stack_data['stack_name']):
                 wait_for(lambda: provider.mgmt.delete_stack(stack_data['stack_name']),
                  delay=10, num_sec=800, message="wait for stack delete")
-            stack_data['vm_name'].delete_from_provider()
```

Comment:
stack_data['vm_name'] is a string, so I don't know what everything after the . does except throw a warning in the log. But yes, it does get cleaned up fine.